crates/astria-composer/src/executor/bundle_factory/mod.rs

```diff
@@ -115,6 +118,14 @@ impl SizedBundle {
 pub(super) enum BundleFactoryError {
     #[error("sequence action is larger than the max bundle size. seq_action size: {size}")]
     SequenceActionTooLarge { size: usize, max_size: usize },
+    #[error(
+        "finished bundle queue is full and the sequence action does not fit in the current bundle"
```

Comment:
The error message should contain all the mentioned quantities, like `curr_bundle_size`, `finished_queue_capacity` etc

Reply:
thanks for catching this